lib/operations/command.rb

```diff
@@ -133,6 +138,7 @@ class OperationFailed < StandardError
   option :contract, Operations::Types.Interface(:call)
   option :policies, Operations::Types::Array.of(Operations::Types.Interface(:call))
   option :preconditions, Operations::Types::Array.of(Operations::Types.Interface(:call)), default: -> { [] }
+  option :idempotency, Operations::Types::Array.of(Operations::Types.Interface(:call)), default: -> { [] }
```

Comment:
option name is in singular form but expects array

Reply:
I think this is fine actually. It is a collective noun in this case. Also, there is no `idempotencies` word

Reply:
ohh, sorry then didn't know that 👍